examples/multiagent/maddpg_iddpg.py

```diff
@@ -170,13 +174,6 @@ def train(cfg: "DictConfig"):  # noqa: F821
 
         sampling_time = time.time() - sampling_start
 
-        tensordict_data.set(
-            ("next", "done"),
-            tensordict_data.get(("next", "done"))
-            .unsqueeze(-1)
-            .expand(tensordict_data.get(("next", env.reward_key)).shape),
-        )  # We need to expand the done to match the reward shape
-
```

Comment:
thanks for this

the same code was used in the multiagent tutorial, I don't know if we want to update that too.
I can do it later if you want

Reply:
How should we do that? Hard to import from examples smth in the tutorials, and re-coding it seems a bit awkward.
To it'd make sense to have that transform in `transforms/multiagent.py` but it's up to you.
Maybe we can make all of this more uniform in a separate PR? I don't feel comfortable enough with this to make the docstrings and tests on my own tbh.